### PR TITLE
minor adjustments

### PR DESCRIPTION
**Pull request by rsill-neo4j**
added a heading (+link to it) for @declareRelationship
and some typos

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-158.surge.sh

Updated pages:
- [Autogeneration](https://neo4j-docs-graphql-158.surge.sh/graphql/5/directives/autogeneration)
- [Custom logic](https://neo4j-docs-graphql-158.surge.sh/graphql/5/directives/custom-logic)
- [Directives](https://neo4j-docs-graphql-158.surge.sh/graphql/5/directives)
- [Global configuration](https://neo4j-docs-graphql-158.surge.sh/graphql/5/directives/schema-configuration/global-configuration)
- [Relationships](https://neo4j-docs-graphql-158.surge.sh/graphql/5/types/relationships)

File: modules/ROOT/pages/directives/autogeneration.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ This page describes directives used for autogeneration:
 This directive marks a field as an identifier for an object type.
 This enables autogeneration of IDs for the field.
 
-The format of each generated ID is a UUID generated by https://neo4j.com/docs/cypher-manual/current/functions/scalar/#functions-randomuuid[randomUUID() function].
+The format of each generated ID is a UUID generated by the link:https://neo4j.com/docs/cypher-manual/current/functions/scalar/#functions-randomuuid[randomUUID() function].
```
Comment (lidiazuin) [inline modules/ROOT/pages/directives/autogeneration.adoc]:
```suggestion
The format of each generated ID is a UUID generated by the link:https://neo4j.com/docs/cypher-manual/current/functions/scalar/#functions-randomuuid[`randomUUID()` function].
```

**Review by lidiazuin — APPROVED** [review]
Apart from a small suggestion, everything looks good to me!

**Review by darrellwarde — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-158.surge.sh

Updated pages:
- [Autogeneration](https://neo4j-docs-graphql-158.surge.sh/graphql/5/directives/autogeneration)
- [Custom logic](https://neo4j-docs-graphql-158.surge.sh/graphql/5/directives/custom-logic)
- [Directives](https://neo4j-docs-graphql-158.surge.sh/graphql/5/directives)
- [Global configuration](https://neo4j-docs-graphql-158.surge.sh/graphql/5/directives/schema-configuration/global-configuration)
- [Relationships](https://neo4j-docs-graphql-158.surge.sh/graphql/5/types/relationships)

**Pull request merged**